carbonmark/components/pages/Users/SellerConnected/Forms/EditListing.tsx

```diff
@@ -155,6 +155,10 @@ export const EditListing: FC<Props> = (props) => {
                     )}`,
                   }),
                 },
+                pattern: {
+                  value: /^\d+(\.\d{1,6})?$/,
+                  message: `The maximum decimals allowed is 6`,
+                },
```

Comment:
This should probably accept both comma and dot as decimal separator (both are supported by `<input type="number">`.

See here:
https://codepen.io/selimslim/pen/OJdwrZX

Reply:
Same as above, comma's don't work at all for me in the codepen, even as a replacement for `.`